Lib/importlib/_bootstrap_external.py

```diff
@@ -463,7 +465,7 @@ def _write_atomic(path, data, mode=0o666):
 # Whenever MAGIC_NUMBER is changed, the ranges in the magic_values array
 # in PC/launcher.c must also be updated.
 
-MAGIC_NUMBER = (3551).to_bytes(2, 'little') + b'\r\n'
+MAGIC_NUMBER = (3554).to_bytes(2, 'little') + b'\r\n'
```

Comment:
I'll set this to the correct value before merging.
The actual number will depend on which PRs get merged first.